ui/css/pods-flexible-relationships.css

```diff
@@ -0,0 +1,24 @@
+#wpadminbar, #adminmenuback, #adminmenuwrap, #preview-action, #wpfooter {
+	display: none;
+}
+
+html.wp-toolbar {
+	padding-top: 0 ;
+}
+
+#wpbody-content {
+	padding-bottom: 0;
+}
+
+.auto-fold #wpcontent, .auto-fold #wpfooter {
+	margin-left: 0;
+}
+
+#wpcontent {
+	padding: 0;
+	margin: 0 1.25em;
+}
+
+.wrap {
+	margin: 0;
+}
```

Comment:
@nicdford @pglewis I know @nicdford already talked about this, but also would like to see padding on the modal content somehow, they're tight on the left.